src/msw/textctrl.cpp

```diff
@@ -339,6 +339,7 @@ void wxTextCtrl::Init()
 {
 #if wxUSE_RICHEDIT
     m_verRichEdit = 0;
+    m_richDPIscale = 0;
```

Comment:
I am not sure if this should be 0 or 1?

Reply:
I set it to 1 now, so I could remove a not-zero check.